tests/testUtils.py

```diff
@@ -441,22 +441,28 @@ def createInitAccounts(self):
         Utils.Print("Publish eosio.system contract")
         trans=self.publishContract(eosio.name, wastFile, abiFile, waitForTransBlock=True)
         if trans is None:
-           Utils.errorExit("Failed to publish oesio.system.")
-
-        Utils.Print("push issue action to eosio contract")
-        contract=eosio.name
-        action="issue"
-        data="{\"to\":\"eosio\",\"quantity\":\"1000000000.0000 EOS\"}"
-        opts="--permission eosio@active"
-        trans=self.pushMessage(contract, action, data, opts)
-        transId=Node.getTransId(trans[1])
-        self.waitForTransIdOnNode(transId)
-
-        expectedAmount=10000000000000
-        Utils.Print("Verify eosio issue, Expected: %d" % (expectedAmount))
-        actualAmount=self.getAccountBalance(eosio.name)
-        if expectedAmount != actualAmount:
-            Utils.errorExit("Issue verification failed. Excepted %d, actual: %d" % (expectedAmount, actualAmount))
+           Utils.errorExit("Failed to publish eosio.system.")
+
+        oldAmount = 0 # oldamount might not be retrievable
+        while True:
+            Utils.Print("push issue action to eosio contract")
+            contract=eosio.name
+            action="issue"
+            data="{\"to\":\"eosio\",\"quantity\":\"1000000000.0000 EOS\"}"
+            opts="--permission eosio@active"
+            trans=self.pushMessage(contract, action, data, opts)
+            transId=Node.getTransId(trans[1])
+            self.waitForTransIdOnNode(transId)
+            
+            expectedAmount=10000000000000+oldAmount
+            Utils.Print("Verify eosio issue, Expected: %d" % (expectedAmount))
+            actualAmount=self.getAccountBalance(eosio.name)
+            if expectedAmount == actualAmount:
+                break
+            if expectedAmount != actualAmount and oldAmount > 0:
+                Utils.errorExit("Issue verification failed. Excepted %d, actual: %d" % (expectedAmount, actualAmount))
+            oldAmount = actualAmount
+            Utils.Print("Try issue again with oldAmount: %d" % (oldAmount))
 
```

Comment:
I feel that you need to bound this loop somehow, maybe a configurable time bound so we don't have an infinite loop.